src/arrow2/src/bitmap/mutable.rs

```diff
@@ -325,6 +325,25 @@ impl MutableBitmap {
     pub(crate) fn bitchunks_exact_mut<T: BitChunk>(&mut self) -> BitChunksExactMut<T> {
         BitChunksExactMut::new(&mut self.buffer, self.length)
     }
+
+    pub fn or(&self, other: &MutableBitmap) -> MutableBitmap {
```

Comment:
You shouldn't need to add `or` here. The key thing is to convert the `MutableBitmap` which is like a builder to `Bitmap` via `.into()` and then use `or` on that. https://github.com/Eventual-Inc/Daft/blob/main/src/arrow2/src/bitmap/bitmap_ops.rs#L179